packages/core/src/lint.ts

```diff
@@ -77,13 +77,35 @@ export async function lintDocument(opts: {
 
   const preprocessors = initRules(rules as any, config, 'preprocessors', oasVersion);
   const regularRules = initRules(rules as Oas3RuleSet[], config, 'rules', oasVersion);
-  const normalizedVisitors = normalizeVisitors([...preprocessors, ...regularRules] as any, types);
+
+  if (config.resolveAfterTransformers) {
```

Comment:
I think this is good to just have this piece copied over. Extracting into function may be tempting but this is easier to read I think.

👍